pages/papers/how-to-practices-and-learn-ruby-tests.mdx

```diff
@@ -4,38 +4,34 @@ lang: en
 description: From beginner to expert in ruby tests.
 ---
 
-<div css={{ color: "red", fontStyle: "italic", fontSize: "2em" }}>
-  In Progress
-</div>
-
 **Tags:** _Ruby_, _Tests_
 
-**Audience Level:** _~~Beginer~~_ _~~All~~_ _Intermediate_
+**Audience Level:** _Beginer_
 
 ### Elevator Pitch (300 characters)
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
 You have 300 characters to sell your talk. This is known as the "elevator pitch". Make it as exciting and enticing as possible.
 </div>
 
-Cuando me inicie en el mundo de la programación los tests fueron un gran punto de partida para reducir mi frustración en el proceso de aprendizaje y coger confianza sobre el código que escribía, ahora en ruby, perfeccione esas habilidades, y te mostrare, como aprender ruby y aprender test! al mismo tiempo!
+When I started in the world of programming the tests were a great starting point to reduce my frustration in the learning process and gain confidence in the code I wrote, now in ruby, perfect those skills, and I will show you how to learn ruby and learn test! at the same time!
 
 ### Description
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
 You should make the description of your talk as compelling and exciting as possible. Remember, you're selling both the organizers of the events to select your talk, as well as trying to convince attendees your talk is the one they should see.
 </div>
 
-La primera cosa alucinante que escribi cuando me inicie en ruby fue un genial y hermoso "Hola mundo", y la segunda cosa aun mas alucinante fue escribir mi primer test para probar que en realidad mi funcion retornaria un "Hola mundo", desde ese entonces no eh parado de escribir pruebas en todo y para todo.
+The first amazing thing I wrote when I started ruby was a great and beautiful "Hello world", and the second thing even more amazing was writing my first test to prove that my function would actually return a "Hello world", since then I have not stopped writing tests in everything and for everything.
 
-Es increible el poder que te da escribir pruebas, esa confianza en uno mismo a la hora de programar, esa sensacion de "Aja! te encontre" (frase mental al encontrar ~~hermosos~~ bugs), ese cambio de paradigma de que ahora estas mas dispuesto a encontrar errores de manera temprana o esa adrenalina que uno siente al hacer correr sus pruebas y el corazon empieza a latir a mil por cada puntito verde que se va coloreando en la consola.
+It is incredible the power that gives you to write tests, that confidence in oneself at the time of programming, that feeling of "Aja! I found you" (mental phrase to find ~~beautiful~~ bugs), that paradigm shift that now you are more willing to find mistakes early or that adrenaline that one feels when running their tests and the heart begins to beat a thousand times for each little green dot that is colored in the console.
```

Comment:
The power writing tests gives you is incredible, that confidence in oneself when programming, that feeling of "Aha! I found you!" [...] mistakes early, or that adrenaline...